storybook/cypress.config.ts

```diff
@@ -5,6 +5,20 @@ export default defineConfig({
   e2e: {
     setupNodeEvents(on, _config) {
       addMatchImageSnapshotPlugin(on);
+      on("task", {
+        log(message) {
+          console.error(message);
+
+          return null;
+        },
+      });
+      on("task", {
+        table(message) {
+          console.error(message);
```

Comment:
Uitzoeken of `console.table` werkt en wellicht een betere kandidaat is.

Reply:
Gedaan